### PR TITLE
Merkl changes - not ready to be merged

### PR DESCRIPTION
**Pull request by tutacrypto**
### What I changed:
- Summary file, and the order of the pages
- names of the pages

### ToDo
- [ ] add screenshots for LP / claim guide
- [x] add screenshot for incentivizor guide
- [ ] add diagram for the integration guide

### CODE REVIEW

File: side-products/merkl/integration-guide.md [hunk 1]
```diff
@@ -20,65 +22,79 @@ The `pools` object also has for each incentivized pool details about the differe
...
+      proofs as string[][]
     )
-  ).wait()
-}
+  ).wait();
+};
 ```
 
+{% hint style="info" %}
+If you want to build the proof yourself, [this code](https://github.com/AngleProtocol/sdk/blob/288185227514ae5c5bb23f5d4b72680eb839f6cc/src/utils/merkl.ts#L15) will show you how to do that.
```
Comment (sogipec) [inline side-products/merkl/integration-guide.md:95]: this file

File: side-products/merkl/integration-guide.md [hunk 2]
```diff
@@ -20,65 +22,79 @@ The `pools` object also has for each incentivized pool details about the differe
...
+On top of displaying how much has been earned by a specific address, the API can also be used to build the payload of a claim transaction of a user which accumulated rewards.
 
-When called for a specific user, it returns in a `transactionData` payload the list of tokens with for each a Merkle proof to use to claim the associated rewards.
+When called for a specific user, it returns in a `transactionData` payload with the list of `tokens` to claim, the `amounts`, and a Merkle `proof` required to claim each of them.
 
-Rewards are claimable per token: meaning that if you have accumulated rewards of several tokens, you may choose to only claim your rewards of one token type, but you may also choose to claim all your token rewards at once.
+_Rewards are claimable per token: meaning that if you have accumulated rewards of several tokens, you may choose to only claim your rewards of one token type, but you may also choose to claim all your token rewards at once._
 
 The contract on which rewards should be claimed is the `Distributor` contract which address can be found on [this docs](helpers.md#🧑‍💻-smart-contracts), or on the [Angle SDK](https://github.com/AngleProtocol/sdk).
```
Comment (sogipec) [inline side-products/merkl/integration-guide.md:39]: make sure that with the reorganization you haven't broken links -> here the link helpers.md may be broken

File: side-products/merkl/integration-guide.md [hunk 3]
```diff
@@ -20,65 +22,79 @@ The `pools` object also has for each incentivized pool details about the differe
...
 
+You have multiple options to do that:
+
+- Rely on Angle's API: we build the claim transaction payload for you and the associated proof, and you just call our API. This is the example shown below.
+- Build the proof yourself and join it to the transaction data from the API. You can find a Github repository below showing how to do that.
```
Comment (sogipec) [inline side-products/merkl/integration-guide.md:44]: Not sure to understand the difference between the two cases -> in the API we're never sending the contract address I think nor building the transaction for you (cf the script below)

Comment (tutacrypto) [reply side-products/merkl/integration-guide.md:44]: Isn't this fetching the contract address from the SDK ? 

CleanShot 2023-04-21 at 15 18 40@2x

Comment (sogipec) [reply side-products/merkl/integration-guide.md:44]: Yes but it does not change anything from the script like -> if you don't update the SDK, address will still be considered valid

**Pull request merged**